include/seqan/sequence/adapt_std_string.h

```diff
@@ -163,6 +163,21 @@ struct DefaultOverflowImplicit< std::basic_string<TChar, TCharTraits, TAlloc> >
 template <typename TChar, typename TCharTraits, typename TAlloc>
 struct IsSequence<std::basic_string<TChar, TCharTraits, TAlloc> > : True {};
 
+// ----------------------------------------------------------------------------
+// Metafunction Chunk
+// ----------------------------------------------------------------------------
+
+// Chunk interface for std::basic strings.
+template <typename TChar, typename TCharTraits, typename TAlloc, typename TSpec>
+struct Chunk<Iter<std::basic_string<TChar, TCharTraits, TAlloc>, AdaptorIterator<TChar*, TSpec> > >
+{
+    typedef typename Chunk<std::basic_string<TChar, TCharTraits, TAlloc> >::Type Type;
+};
```

Comment:
This will work, but is not very efficient. ```Chunk<std::basic_string<TChar, TCharTraits, TAlloc> >::Type``` will evaluate to ```Nothing``` and disables chunking for the std::string. Instead the slower appendValue will be used. It is fine for now, but we should leave the corresponding bug report open and put a remark there to actually provide real chunking for std::string.
I think many of the functionality is already covered by the chunking support for the ```Iter<TContainer, AdaptorIterator<TValue*, TSpec> > ``` in basic/iterator_adaptor.h but it seems that some functions behave different or are not used for std::string.